### PR TITLE
W-15760813 munit settings duke

### PR DESCRIPTION
**Pull request by dukesphere**

### CODE REVIEW

File: modules/ROOT/pages/int-test-munit.adoc [hunk 1]
```diff
@@ -41,16 +41,53 @@ Anypoint Code Builder - MUnit Extension, part of the Anypoint Extension Pack, su
...
 
-Understand the fundamentals of MUnit. Refer to the MUnit documentation for an overview and guidance with test configurations:
+* Understand the fundamentals of MUnit. Refer to the MUnit documentation for an overview and guidance with test configurations:
+
+** xref:munit::index.adoc[MUnit Overview]
+** xref:munit::munit-test-concept.adoc[]
+** xref:munit::event-processors.adoc[]
+** xref:munit::munit-cookbook.adoc[]
+
+* You must have an enterprise license to configure your Maven `settings.xml`. See <<maven-settings>>.
```
Comment (dukesphere) [inline modules/ROOT/pages/int-test-munit.adoc]:
```suggestion
* You must have an enterprise license to access MUnit resources through your Maven `settings.xml` file. See <<maven-settings>>.
```

File: modules/ROOT/pages/_partials/acb-maven-settings.adoc [hunk 2]
```diff
@@ -19,3 +19,24 @@ Updates that you save to your `settings.xml` are reflected automatically within
...
+
+//
+// tag::nav-to-maven-settings[]
+Navigate to `settings.xml`, typically in `{user.home}/.m2/`.
+// end::nav-to-maven-settings[]
+//
+
+//
+// tag::add-plugin-repo-settings[]
+Add the following `pluginRepository` setting _within_ `<profile><pluginRepositories></pluginRepositories></profile>`:
```
Comment (egrave) [inline modules/ROOT/pages/_partials/acb-maven-settings.adoc]: EE repository should be declared only as Repository, not as PluginRepository

File: modules/ROOT/pages/int-test-munit.adoc [hunk 3]
```diff
@@ -41,23 +41,92 @@ Anypoint Code Builder - MUnit Extension, part of the Anypoint Extension Pack, su
...
+  <username>yourMuleUsername</username>
+  <password>yourMulePassword</password>
+  <id>releases-ee</id>
+</server>
+----
+
+[[configure-settings-license-no]]
+=== Configure Maven Settings (Unlicensed Access)
+
+If you do not have a Mule EE license or if your license is a trial license, configure your `mule-artifact.json` for the project and `pluginRepository` settings in the Maven file, `settings.xml`.
```
Comment (egrave) [inline modules/ROOT/pages/int-test-munit.adoc]: No settings are required when using community edition (just the change in mule-artifact.json is enough)

**Review by egrave — DISMISSED** [review]

File: modules/ROOT/pages/int-test-munit.adoc [hunk 4]
```diff
@@ -41,23 +41,84 @@ Anypoint Code Builder - MUnit Extension, part of the Anypoint Extension Pack, su
...
 
-//
 For MUnit features supported by Anypoint Code Builder, see xref:acb-release-notes.adoc[]. Anypoint Studio supports testing with all MUnit features.
 
+//
+//MUNIT SETTINGS
+[[maven-settings]]
+== Configure Access to MUnit Features
+
+Configure your project to connect to MUnit resources. The process differs depending on whether you have a Mule enterprise edition (EE) license.
```
Comment (dukesphere) [inline modules/ROOT/pages/int-test-munit.adoc]:
```suggestion
Make MUnit resources available to your project. The process differs depending on whether you have a Mule enterprise edition (EE) license.
```

File: modules/ROOT/pages/_partials/acb-maven-settings.adoc (hunk 2 above)
Comment (dukesphere) [reply modules/ROOT/pages/_partials/acb-maven-settings.adoc]: done

File: modules/ROOT/pages/int-test-munit.adoc (hunk 3 above)
Comment (dukesphere) [reply modules/ROOT/pages/int-test-munit.adoc]: done

File: modules/ROOT/pages/int-test-munit.adoc [hunk 5]
```diff
@@ -41,23 +41,88 @@ Anypoint Code Builder - MUnit Extension, part of the Anypoint Extension Pack, su
...
+  <username>yourMuleUsername</username>
+  <password>yourMulePassword</password>
+  <id>releases-ee</id>
+</server>
+----
+
+[[configure-settings-license-no]]
+=== Configure Maven Settings (Unlicensed Access)
+
+If you do not own a Mule EE license or if your license is a trial license, configure your `mule-artifact.json` for the project.
```
Comment (egrave) [inline modules/ROOT/pages/int-test-munit.adoc]: I'd write something like:"...configure your `mule-artifact.json` for the project  to use mule community edition

File: modules/ROOT/pages/int-test-munit.adoc [hunk 6]
```diff
@@ -41,23 +41,88 @@ Anypoint Code Builder - MUnit Extension, part of the Anypoint Extension Pack, su
...
+
+See xref:mule-runtime::installing-an-enterprise-license.adoc[Install an Enterprise License], to acquire or install an Enterprise license. If you are a MuleSoft customer and do not have credentials to access the https://repository.mulesoft.org/nexus-ee/content/repositories/releases-ee/[MuleSoft Enterprise repository^], contact Support and request Nexus enterprise credentials.
+
+//navigate to settings.xml:
+. {empty}
+include::anypoint-code-builder::partial$acb-maven-settings.adoc[tag=nav-to-maven-settings]
+. Specify `server` settings with your Mule username and password _within_ `<servers></servers>`:
++
+[source,xml]
+----
```
Comment (egrave) [inline modules/ROOT/pages/int-test-munit.adoc:89]: mule-ee repository should be added as "repository" too:
```
        <repository>
          <id>releases-ee</id>
          <name>Mule Release Repository</name>
          <url>https://repository-master.mulesoft.org/nexus/content/repositories/releases-ee</url>
        </repository>
```

Comment (dukesphere) [reply modules/ROOT/pages/int-test-munit.adoc:89]: done

**Review by Crispy-Salesforce — APPROVED** [review]

**Pull request merged**